src/DependencyInjection/Providers/AzureProviderConfigurator.php

```diff
@@ -35,21 +39,26 @@ public function buildConfiguration(NodeBuilder $node)
             ->scalarNode('tenant')
                 ->example("tenant: 'common'")
                 ->info('The tenant to use, default is `common`')
+                ->defaultValue('common')
             ->end()
             ->scalarNode('url_api')
                 ->example("url_api: 'https://graph.windows.net/'")
                 ->info('Domain to build request URL')
+                ->defaultValue('https://graph.windows.net/')
             ->end()
             ->scalarNode('resource')
                 ->info('Oauth resource field')
+                ->defaultValue('')
             ->end()
             ->scalarNode('api_version')
                 ->example("api_version: '1.6'")
                 ->info('The API version to run against')
+                ->defaultValue('1.6')
```

Comment:
I think we should avoid this. The problem is that setting a default API version means we may need to "bump" it up in the future. Also, making the user choose is a good idea - then they will go find the correct version of the docs for that version.

Reply:
I concur, with reference to "we may need to "bump" it up". I wouldn't want to be responsible for watching the provider to see if they change their default API version.

That being said, we could run into the same issue with any of these configuration options. I mean, the odds of Microsoft making significant changes to its default paths, login url, graph url, etc are very slim. I doubt they want to break BC that bad, but it could happen and provider would end up needing updated leading us to have to update it here. 

API version I can see being a problem; However, the initial commit was November 16, 2015 and it was 1.6 then as well, so idk. I might push back a bit on the `resource` empty string one above, but this one -- just tell me what to do and I'll do it.

Reply:
I updated the `'api_version'` node to remove the default value altogether